elyra/tests/pipeline/kfp/test_processor_kfp.py

```diff
@@ -992,7 +992,7 @@ def test_generate_pipeline_dsl_compile_pipeline_dsl_one_generic_node_pipeline_te
     monkeypatch.setattr(processor, "_upload_dependencies_to_object_store", lambda w, x, y, prefix: True)
     monkeypatch.setattr(processor, "_verify_cos_connectivity", lambda x: True)
 
-    compiled_argo_output_file = Path(tmpdir) / test_pipeline_file.with_suffix(".yaml")
+    compiled_argo_output_file = Path(tmpdir) / os.path.basename(test_pipeline_file.with_suffix(".yaml"))
```

Comment:
Nice catch! Would you mind switching to the pathlib-based approach? For the past couple of months I've been replacing `os.path` to their more modern equivalent. There should be seven `test_pipeline_file.with_suffix` in this file. (Next time I need to touch the parser tests I will also replace the `tmpdir` fixture in all tests with the `tmp_path` fixture, which yields a path object instead of a string.)

```suggestion
    compiled_argo_output_file = Path(tmpdir) / test_pipeline_file.with_suffix(".yaml").name
```

Reply:
Sounds good - thanks.  Btw, I'm only finding five of these instances.

Reply:
oops, I can't count